hoodie-utilities/src/main/java/com/uber/hoodie/utilities/HoodieCompactorAdmin.java

```diff
@@ -0,0 +1,154 @@
+package com.uber.hoodie.utilities;
+
+import com.beust.jcommander.JCommander;
+import com.beust.jcommander.Parameter;
+import com.uber.hoodie.CompactionAdminClient;
+import com.uber.hoodie.CompactionAdminClient.RenameOpResult;
+import com.uber.hoodie.CompactionAdminClient.ValidationOpResult;
+import com.uber.hoodie.common.table.HoodieTableMetaClient;
+import com.uber.hoodie.common.util.FSUtils;
+import java.io.ObjectOutputStream;
+import java.io.Serializable;
+import java.util.List;
+import org.apache.hadoop.fs.FSDataOutputStream;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+import org.apache.spark.api.java.JavaSparkContext;
+
+public class HoodieCompactorAdmin {
+
+  private final Config cfg;
+
+  public HoodieCompactorAdmin(Config cfg) {
```

Comment:
rename to "HoodieCompactionDriver" ?

Reply:
or `HoodieCompactionTool`. Something that calls out this is an utility to run and not an internal class

Reply:
is this the class one should consider to be scheduled via a Workflow manager ? I am wondering if we need a separate class or we can just run a series of hoodie-cli commands instead passed into stdin?

Reply:
I will rename to HoodieCompactionAdminTool. This class only contain methods to unschedule and repair compactions - which requires manual triggering and guarantee that no concurrent compaction/ingestion is happening. The general compaction triggering methods - compaction schedule and compaction run are in the original HoodieCompactor class. 

I kept the pattern is similar to other Hoodie-CLI commands (e:g - hdfsparquetimport) which involves running Spark-Launcher process. 
There are 2 options to run - One to invoke from Hoodie-CLI and another to run directly as spark-submit.  Agree that this class must not be scheduled as part of Workflow-Manager. But, for whatever reasons, if someone want to submit this as spark-submit job, its possible with this approach.